selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/CreateNewPackagesWithHelpCreationJavaClassTest.java

```diff
@@ -52,7 +53,7 @@
   @Inject private TestProjectServiceClient testProjectServiceClient;
   @Inject private HttpJsonRequestFactory httpJsonRequestFactory;
   @Inject private TestApiEndpointUrlProvider testApiEndpointUrlProvider;
-  @Inject private BrowserLogsUtil browserLogsUtil;
+  @Inject private SeleniumWebDriver seleniumWebDriver;
```

Comment:
it's better to avoid injection of WebDriver into the test. Lets inject webdriver in the same way as it was done in **SeleniumWebDriverHelper** class.